README.md

```diff
@@ -3,13 +3,29 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  Things like lower order terms may be ignored in asymptotic analysis, but obviously wouldn't be ignored in practice as everything is used. Even just this one example leads to more inputs in practice.
+
+  Optimization of the code's use of hardware always has an impact on the performance of the code. If it is not optimized properly to utilize resources efficiently, the code may run slow in practice. For example, sometimes things like a GPU can be used in hardware acceleration to help optimize a program.
+
+  The third reason is that asymptotic analysis is theoretical. You need to actually test it to get real data, as there may be differences based on things such as inputs alone, or unexpected issues that the user may come across.
```

Comment:
Can you be more specific here please? It seems that this is just summarizing the previous reasons.